src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -81,60 +82,21 @@ private enum GetAncestorFlags
         /// <returns>A <see cref="Task"/> of <see cref="TokenResult"/>.</returns>
         public async Task<AuthFlowResult> GetTokenAsync()
         {
-            IAccount account = await this.pcaWrapper.TryToGetCachedAccountAsync(this.preferredDomain)
-                 ?? PublicClientApplication.OperatingSystemAccount;
+            IAccount account = await this.GetCachedAccountAsync().ConfigureAwait(false);

```

Comment:
@goagain we do not want to cahnge this AuthFlow. Broker Always has an account beacuse of the `?? PublicClientApplication.OperatingSystemAccount;` - this is very important and is how on Corp machines, we default to silently authenticating.

Reply:
I moved the logic to `GetCachedAccountAsync`. I want to match the interface's definition.